oio/xcute/orchestrator.py

```diff
@@ -510,8 +510,6 @@ def _check_beanstalkd_worker(self, beanstalkd_info):
         if not beanstalkd:
             beanstalkd = Beanstalk.from_url(beanstalkd_addr)
             beanstalkd.addr = beanstalkd_addr
-            beanstalkd.use(self.beanstalkd_workers_tube)
-            beanstalkd.watch(self.beanstalkd_workers_tube)
```

Comment:
Indeed, the use of the tube requested too early. But these methods must still be called to specifically use this tube. We could do it after all the checks (https://github.com/open-io/oio-sds/pull/1981/files#diff-965c127dbd1971b4f3ad28a7884213eaL544).

Reply:
My bad, for some reason, without doing `use`, a job is still dispatched without raising an error (that's weird). But why do we need the `watch`? We're not reading from the tube.